exporter/kineticaexporter/metrics_exporter.go

```diff
@@ -0,0 +1,94 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kineticaexporter
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+)
+
+type kineticaMetricsExporter struct {
+	logger *zap.Logger
+}
+
+func newMetricsExporter(_ *zap.Logger, cfg *Config) (*kineticaMetricsExporter, error) {
+	if err := cfg.Validate(); err != nil {
+		return nil, err
+	}
+
+	kineticaLogger := cfg.createLogger()
+	metricsExp := &kineticaMetricsExporter{
+		logger: kineticaLogger,
+	}
+	return metricsExp, nil
+}
+
+func (e *kineticaMetricsExporter) start(_ context.Context, _ component.Host) error {
+
+	return nil
+}
+
+// shutdown will shut down the exporter.
+func (e *kineticaMetricsExporter) shutdown(_ context.Context) error {
+	return nil
+}
+
+// pushMetricsData - this method is called by the collector to feed the metrics data to the exporter
+//
+//	@receiver e
+//	@param _ ctx unused
+//	@param md
+//	@return error
+func (e *kineticaMetricsExporter) pushMetricsData(_ context.Context, md pmetric.Metrics) error {
```

Comment:
For all signals, please remove all business logic around exporting.  This PR should only be the factory and config. You'll need these functions to satisfy the factory, but for now it only return nil.

Reply:
Removed all business logic as suggested and the methods return `nil` for now.